lib/streamlit/elements/time_widgets.py

```diff
@@ -222,6 +222,7 @@ def time_input(
         *,  # keyword-only arguments:
         disabled: bool = False,
         label_visibility: LabelVisibility = "visible",
+        step: Union[int, timedelta] = timedelta(minutes=15),
```

Comment:
I'd extract `15` to the constant, for example `DEFAULT_STEP_MINUTES=15`

Reply:
Good point, it was repeated in two places, so I've extracted it to constants 👍